From: Shengjing Zhu <i@zhsj.me>
Date: Sun, 8 Jul 2018 15:05:26 +0800
Subject: use python3 for emugl headers generate scripts

Signed-off-by: Shengjing Zhu <i@zhsj.me>
---
 scripts/gen-emugl-entries.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/gen-emugl-entries.py b/scripts/gen-emugl-entries.py
index b88137e..73ede51 100755
--- a/scripts/gen-emugl-entries.py
+++ b/scripts/gen-emugl-entries.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
 
 # Copyright 2015 The Android Open Source Project
 #
