From 32cb7f26c297793c177ae4ddb66e95fe32746544 Mon Sep 17 00:00:00 2001
From: Shengjing Zhu <i@zhsj.me>
Date: Sun, 27 Oct 2019 22:40:58 +0800
Subject: [PATCH] Fix gmock lib path

---
 cmake/FindGMock.cmake | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/cmake/FindGMock.cmake b/cmake/FindGMock.cmake
index 21f5cdb..a1f7644 100644
--- a/cmake/FindGMock.cmake
+++ b/cmake/FindGMock.cmake
@@ -47,10 +47,10 @@ endif()
 set(findgmock_cxx_flags "${CMAKE_CXX_FLAGS} -g -Wno-old-style-cast -Wno-missing-field-initializers -Wno-ctor-dtor-privacy -Wno-switch-default")
 
 set(findgmock_bin_dir "${CMAKE_CURRENT_BINARY_DIR}/gmock")
-set(findgmock_gtest_lib "${findgmock_bin_dir}/gtest/libgtest.a")
-set(findgmock_gtest_main_lib "${findgmock_bin_dir}/gtest/libgtest_main.a")
-set(findgmock_gmock_lib "${findgmock_bin_dir}/libgmock.a")
-set(findgmock_gmock_main_lib "${findgmock_bin_dir}/libgmock_main.a")
+set(findgmock_gtest_lib "${findgmock_bin_dir}/lib/libgtest.a")
+set(findgmock_gtest_main_lib "${findgmock_bin_dir}/lib/libgtest_main.a")
+set(findgmock_gmock_lib "${findgmock_bin_dir}/lib/libgmock.a")
+set(findgmock_gmock_main_lib "${findgmock_bin_dir}/lib/libgmock_main.a")
 
 include(ExternalProject)
 ExternalProject_Add(GMock SOURCE_DIR "${GMOCK_SOURCE_DIR}"
-- 
2.23.0

